### ISSUE TITLE
The code definition of the custom loss function.

### ISSUE THREAD

**Issue by upup-721** [LianpoWang/MaskDICNet#33]
In which file is the code definition of the custom loss function in this repository located? What is the code binding method between the loss function and the optimizer during training?
